Sources/SwiftGen/Commander/ConfigCommands.swift

```diff
@@ -54,58 +54,60 @@ extension ConfigEntry {
 
 // MARK: - Commands
 
-private let configOption = Option<Path>(
-  "config",
-  default: "swiftgen.yml",
-  flag: "c",
-  description: "Path to the configuration file to use",
-  validator: checkPath(type: "config file") { $0.isFile }
-)
+enum ConfigCLI {
+  private static let configOption = Option<Path>(
```

Comment:
I'd do the same here as you did below for ParserCLI, and move this inside an `Option` enum.